runctx/runctx.go

```diff
@@ -32,7 +32,8 @@ type Group struct {
 
 func NewGroup(fn ...func(ctx context.Context) error) *Group {
```

Comment:
I thinking of renaming it to `DefaultSignalsGroup`. WDYT @mmatczuk?

Reply:
Nah